### DISCUSSION TITLE
Contributors Counter Component Thingy

### DISCUSSION THREAD

**Discussion by outslept** [fuma-nama/fumadocs#515] [General]
Hey, wasn't sure where to write about it. So you've mentioned once in a PR that adding this type of component would be a great idea. I've played a lil and got one (not styled for now, yet fully functional and fully flexible. It makes a fetch request to Github API, gets the contributors list and filters [bot] guys out of it and shows numbers of contributors without the [bot] guys as well as 4 random contributors avatars). I'd like to ask you a few questions

1) Firstly about the UI, how do you see this component? Is it still the same vercel-ish minimalism? Honestly, I can't find any reference for myself. I would be glad if you share your thoughts on this.

2) What about solutions for fetch? Do you wish to use useEffect or maybe some thing like SWR or smth? I implemented the thing without third-party deps, however, I can rewrite the thing to needed tech if needed :/

I'll also attach for clarity how it looks unstylised.

![counter thingy](https://github.com/fuma-nama/fumadocs/assets/135520429/a2bd27e4-e5c9-490b-b4ab-e46988735969)

**Comment by fuma-nama** [comment]
Thank you so much! It's good enough, I'll handle the UI design part later. I would prefer SWR since it's also used in Fumadocs, could you change it?

**Reply by outslept** [reply]
> Thank you so much! It's good enough, I'll handle the UI design part later. I would prefer SWR since it's also used in Fumadocs, could you change it?

Absolutely! I'll open a PR as soon as I will be free today.

**Comment by fuma-nama** [comment]
It is now available on our landing page, thanks to @changeelog ;)
